src/java/org/apache/cassandra/tools/nodetool/UpgradeSSTable.java

```diff
@@ -56,13 +59,25 @@ public void execute(NodeProbe probe)
 
         for (String keyspace : keyspaces)
         {
-            try
-            {
-                probe.upgradeSSTables(probe.output().out, keyspace, !includeAll, maxSSTableTimestamp, jobs, tableNames);
-            }
-            catch (Exception e)
+            for (int retries = 0; retries < 5; retries++)
             {
-                throw new RuntimeException("Error occurred during enabling auto-compaction", e);
+                try
+                {
+                    if (retries > 0)
+                        Thread.sleep(500);
```

Comment:
use Uninterruptible sleep here maybe?

Reply:
:shrug: I don't think we care being interrupted or not here. If we do, so be it imo?